torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -309,12 +339,19 @@ StaticRuntime::IndividualMetrics StaticRuntime::benchmark_individual_ops(
 
   // main runs
   for (int i = 0; i < main_runs; i++) {
+    if (planner_) {
```

Comment:
what about a default `TrivialMemoryPlanner` that basically no-ops the `allocate_memory` call?  it would require a bit of a refactor for the `deallocate_registers` so let's not do it in this PR, but I'm curious your opinion on the suggestion

Reply:
Why do you need to do that?